cfme/cloud/keypairs.py

```diff
@@ -0,0 +1,34 @@
+
+import cfme.fixtures.pytest_selenium as sel
+from cfme.web_ui import Quadicon, Region, flash
+from utils.pretty import Pretty
+from functools import partial
+from cfme.web_ui import toolbar as tb
+from utils.wait import wait_for
+
+
+details_page = Region(infoblock_type='detail')
+cfg_btn = partial(tb.select, "Configuration")
+
+
+class KeyPairs(Pretty):
```

Comment:
The instances of the class represent only one keypair so the class should be named `Keypair` - singular.